be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandlerTest.scala

```diff
@@ -27,31 +29,52 @@ class RollCallHandlerTest extends TestKit(ActorSystem("RollCall-DB-System")) wit
     val dbActorMock = Props(new Actor() {
       override def receive: Receive = {
         // You can modify the following match case to include more args, names...
-        case m: DbActor.WriteAndPropagate =>
-          system.log.info("Received {}", m)
+        case DbActor.WriteAndPropagate(_, _) | DbActor.ChannelExists(_) | DbActor.CreateChannel(_, _) =>
+          system.log.info(f"Received a message")
           system.log.info("Responding with a Nack")
-
           sender() ! Status.Failure(DbActorNAckException(1, "error"))
+        case x =>
+          system.log.info(s"Received - error $x")
       }
     })
     system.actorOf(dbActorMock, "MockedDB-NACK")
   }
 
-  def mockDbWithAck: AskableActorRef = {
+  def mockDbRollCallNotCreated: AskableActorRef = {
     val dbActorMock = Props(new Actor() {
       override def receive: Receive = {
         // You can modify the following match case to include more args, names...
-        case m: DbActor.WriteAndPropagate =>
-          system.log.info("Received {}", m)
+        case DbActor.WriteAndPropagate(_, _) | DbActor.CreateChannel(_, _) =>
+          system.log.info(s"Received a message")
```

Comment:
```suggestion
        case m@(DbActor.WriteAndPropagate(_, _) | DbActor.CreateChannel(_, _)) =>
          system.log.info("Received {}", m)
```
Instead of downgrading the log message. (same at line 33)